### ISSUE TITLE
Training get stuck in iteration 660

### ISSUE THREAD

**Issue by leefionglee** [rbgirshick/py-faster-rcnn#137]
Hi All,

I met one problem where I trained py-faster-rcnn on my own dataset, but it get stuck in iteration 660 for a long time and no response.  Has anyone met this problem before? How to solve it?  Thanks a lot!

FYR:

I0404 17:50:23.514286 25601 solver.cpp:571] Iteration 640, lr = 0.001
I0404 17:50:24.876749 25601 solver.cpp:242] Iteration 660, loss = 0.341694
I0404 17:50:24.876804 25601 solver.cpp:258]     Train net output #0: rpn_cls_loss = 0.296529 (* 1 = 0.296529 loss)
I0404 17:50:24.876819 25601 solver.cpp:258]     Train net output #1: rpn_loss_bbox = 0.0451643 (* 1 = 0.0451643 loss)
I0404 17:50:24.876835 25601 solver.cpp:571] Iteration 660, lr = 0.001

**Comment by leefionglee** [comment]
Does anyone know why this happened?  Thanks!

**Comment by geng-m** [comment]
Hi,are you figure out this problem? l also has this bug in my project，but this bug occur in diff iterations, such as 200+,5000+,10000+?
